.github/workflows/hpu-gaudi2.yml

```diff
@@ -39,13 +39,14 @@ jobs:
     # The type of runner that the job will run on
     runs-on: [self-hosted, intel, gaudi2]
     container:
-      image: vault.habana.ai/gaudi-docker/1.15.1/ubuntu22.04/habanalabs/pytorch-installer-2.2.0:latest
+      image: vault.habana.ai/gaudi-docker/1.16.0/ubuntu22.04/habanalabs/pytorch-installer-2.2.2:latest
```

Comment:
Looks like that is hanging, so I'll close that PR, but this was passing before with your previous changes, what other changes do you think we need?

Reply:
@loadams ,

This release version upgrade requires updating the driver version on the node first. I am checking on this. Lets wait until latest driver version upgraded at Node.

Reply:
Thanks for the update, I've closed that PR.  But let us know if we need to re-create the runner or how we can update the driver version on the node, since I think this is in our account at Intel Cloud.